src/mount/acquired_files_last_time_used.h

```diff
@@ -0,0 +1,96 @@
+/*
+   Copyright 2013-2017 Skytechnology sp. z o.o.
+   Copyright 2023      Leil Storage OÜ
+
+   This file is part of SaunaFS.
+
+   SaunaFS is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, version 3.
+
+   SaunaFS is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with SaunaFS. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "common/platform.h"
+
+#include <ctime>
+#include <map>
+#include <mutex>
+#include <set>
+#include <stdexcept>
+
+inline uint32_t gCleanAcquiredFilesPeriod;
+inline uint32_t gCleanAcquiredFilesTimeout;
+
+// Define the AcquiredFileLastTimeUsed struct
+struct AcquiredFileLastTimeUsed {
```

Comment:
I think suffix `LastTimeUsed` for the struct can be confusing and is not needed. I think is enough by having the attribute `lastTimeUsed`. The current name is confusing with the container, which also uses a very similar name.
IMHO, `AcquiredFile` is correct as data type.

Reply:
In this case I considered to set it as this because the definition of `AcquiredFile` should contain an integer representing the inode of the file and a number of references to it, which is not currently specified as a data type, but is implicitly defined. In that case I wanted to remark the difference because on the other side, there is an `AcquiredFiles` container that basically contains a map with inode as a key and a number of references to it as a value, which implicitly could define a type, so for differentiating from this, I decided to make a different data type definition.